Userland/Libraries/LibSoftGPU/Config.h

```diff
@@ -24,6 +24,14 @@ static constexpr int MAX_TEXTURE_SIZE = 2048;
 static constexpr float MAX_TEXTURE_LOD_BIAS = 2.f;
 static constexpr int SUBPIXEL_BITS = 4;
 
+static constexpr int NUM_SHADER_INPUTS = 64;
+
+// Verify that we have enough inputs to hold vertex color and texture coordinates for all fixed function texture units
+static_assert(NUM_SHADER_INPUTS >= 4 + GPU::NUM_TEXTURE_UNITS * 4);
+
+static constexpr int SHADER_INPUT_VERTEX_COLOR = 0;
+static constexpr int SHADER_INPUT_FIRST_TEXCOORD = 4;
```

Comment:
It might be nicer to have these indices inside of an `enum` instead of constants; we'll probably use them alot

Reply:
Right, that makes sense